addons/missileguidance/functions/fnc_javelin_midCourseTransition.sqf

```diff
@@ -19,6 +19,10 @@
 #define STAGE_COAST 3
 #define STAGE_TERMINAL 4
 
+#define REQUIRED_STABLE_TIME 0.2
```

Comment:
the stability time is because switching between direct and ZEM will make the flight behaviour funky, usually coming in too low. If we wait until stable, ZEM will be able to do it's thing